Src/Plugins/Tests.Security/Validators/SEC101_044.NpmCredentialsValidatorTests.cs

```diff
@@ -35,43 +38,88 @@ public void NpmCredentialsValidator_MockHttp()
 
             string fingerprintText = $"[host={host}][id={id}][secret={secret}]";
 
+            var authorizedResponse = new HttpResponseMessage(HttpStatusCode.OK);
```

Comment:
>HttpResponseMessage(HttpStatusCode.OK); [](http://example.com/codeflow?start=41&length=39)

These objects look very general purpose, i.e., a response that returns OK, or Unauthorized. We should consider pushing them directly into the HttpMockHelper so that others can reuse them.

Reply:
done!
thanks for the suggestion.
also cleared this with the re-usable HttpResponseMessages